src/io/matpower.jl

```diff
@@ -365,6 +368,27 @@ function matpower_to_powermodels(mp_data::Dict{String,Any})
 end
 
 
+"""
+    split_loads_shunts(data)
+
+Seperates Loads and Shunts in `data` under separate "load" and "shunt" keys in the
+PowerModels data format. Includes references to originating bus via "load_bus"
+and "shunt_bus" keys, respectively.
+"""
+function split_loads_shunts(data::Dict{String,Any})
+    data["load"] = []
+    data["shunt"] = []
+
+    for (i,bus) in enumerate(data["bus"])
+        append!(data["load"], [Dict{String,Any}("pd" => bus["pd"], "qd" => bus["qd"], "load_bus" => bus["bus_i"], "bus_type" => bus["bus_type"], "index" => bus["index"])])
+        append!(data["shunt"], [Dict{String,Any}("gs" => bus["gs"], "bs" => bus["bs"], "shunt_bus" => bus["bus_i"], "bus_type" => bus["bus_type"], "index" => bus["index"])])
+
```

Comment:
Because Matpower does not support omitting loads or shunts, we should not add load if `pd == 0.0 and qd == 0.0` and we should not add a shunt if `gs == 0.0 and bs == 0.0`.  The indexes for each type of component should go from 1 to n, based how many occur in the file with this skipping rule.

Also, this,
```
"bus_type" => bus["bus_type"]
```
should be something like,
```
"status" => bus["bus_type"] != 4
```
Bus status `4` means disconnected from the network.